src/settings/BatchGroupConfigurationSettings/BatchGroupCongiurationSettingsForm.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import stripesFinalForm from '@folio/stripes/final-form';
+import {
+  Button,
+  KeyValue,
+  Pane,
+  PaneFooter,
+  Select,
+} from '@folio/stripes/components';
+
+const BatchGroupConfigurationSettingsForm = ({
+  batchGroups,
+  batchGroupsOptions,
+  handleSubmit,
+  paneTitle,
+  pristine,
+  setBatchGroupId,
+  submitting,
+}) => {
+  const getPaneFooter = () => {
+    const start = (
+      <Button
+        buttonStyle="default mega"
+        data-test-run-manual-export-button
+        disabled
+      >
+        <FormattedMessage id="ui-invoice.button.runManualExport" />
+      </Button>
+    );
+
+    const end = (
+      <Button
+        buttonStyle="primary mega"
+        data-test-save-button
+        disabled={pristine || submitting}
+        onClick={handleSubmit}
+        type="submit"
+      >
+        <FormattedMessage id="ui-invoice.button.save" />
+      </Button>
+    );
+
+    return (
+      <PaneFooter
+        renderStart={start}
+        renderEnd={end}
+      />
+    );
+  };
+
+  return (
+    <Pane
+      data-test-batch-group-configuration-settings
+      defaultWidth="fill"
+      footer={getPaneFooter()}
+      paneTitle={paneTitle}
+    >
+      {batchGroups.length > 1
```

Comment:
this Select and KeyValue looks like a solid component to me. Also I tried to find form and no luck, but the PR's title says it's about `settings form`. What am I missing?

Reply:
Form fields will be added in [UINV-103](https://issues.folio.org/browse/UINV-103). Select and KeyValue are out of the form.